strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -98,6 +222,20 @@ def prepare_displaced_squeezed_state(self, r_d, phi_d, r_s, phi_s, mode):
         self.circuit.squeeze(r_s, phi_s, mode)
         self.circuit.displace(r_d, phi_d, mode)
 
+    def prepare_cat(self, alpha, phi, desc):
+        """ Prepares the arrays of weights, means and covs for a cat state"""
+
```

Comment:
why some `prepare_*` functions have pass and other do not?

Reply:
Made all empty prep classes return nothing